.travis.yml

```diff
@@ -1,8 +1,33 @@
 sudo: false
 language: python
 python:
-    - "3.5"
-    - "3.6"
-    - "3.7"
+- '3.5'
+- '3.6'
+- '3.7'
 install: pip install tox-travis
 script: tox
+before_deploy:
+- python setup.py sdist
+- python setup.py bdist_wheel
+deploy:
+- provider: releases
+  skip_cleanup: true
+  file: dist/*
+  file_glob: true
+  api_key:
```

Comment:
Hmm I assume this is travis-encrypted?

Reply:
yup these are both encrypted keys that are write-scoped